<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addCourse">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Course</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="courseList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>179</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="scrollLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Courses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startTracking">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Tracking (Will notify when slots open)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="term">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>fall</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>winter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>spring</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="trackType">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Open Seats</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Open Waitlist</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Season/Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of Tracking</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CRN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CRN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
